boards/riscv/rv32m1_vega/Kconfig.defconfig

```diff
@@ -91,6 +91,10 @@ if BT
 config BT_CTLR
 	default y
 
+config HAS_RV32M1_CAU3_BLE
+	bool
+	default y if BT_CTLR_CRYPTO && BT_CTLR_PRIVACY_SUPPORT
```

Comment:
default y if BT_CTLR_CRYPTO **||** BT_CTLR_PRIVACY_SUPPORT
?

CTLR_CRYPTO is enabled by default for BT_CTLR

 CTLR_CRYPTO and CTLR_PRIVACY_SUPPORTare not directly related

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/22589#discussion_r377211802